app/resources/serializable_configuration_record.rb

```diff
@@ -0,0 +1,19 @@
+class SerializableConfigurationRecord < JSONAPI::Serializable::Resource
+  type 'configuration'
+
+  attribute :'api-key' do
+    @object.api_key
+  end
+
+  attribute :'customer-id' do
+    @object.customer_id
+  end
```

Comment:
wish there was a better way to do this but i haven't found one yet.  if there isn't, that's one strike against `jsonapi-rails` in my book, considering the json-api definition [recommends](http://jsonapi.org/recommendations/) dashes

Reply:
I agree, that stinks.